Source/KSCrash/Source/KSCrash/Recording/BSG_KSCrashReport.c

```diff
@@ -2182,7 +2182,11 @@ void bsg_kscrashreport_writeStandardReport(BSG_KSCrash_Context* const crashConte
                                 crashContext->config.crashID,
                                 crashContext->config.processName);
 
-        bsg_kscrw_i_writeBinaryImages(writer, BSG_KSCrashField_BinaryImages);
+        // Don't write the binary images for user reported crashes to improve performance
+        if (crashContext->crash.threadTracingEnabled == true ||
+            crashContext->crash.crashType != BSG_KSCrashTypeUserReported) {
```

Comment:
Disabling writing binary images will need to be behind a flag for unity -- we still need binary images for user reported issues on regular cocoa.